### ISSUE TITLE
Equivalent width incorrect

### ISSUE THREAD

**Issue by hcferguson** [spacetelescope/specviz#223]
Looks like the equivalent width calculation is incorrect. I'm getting values around 1.e23 on a feature with a hand-measured EW ~ 94. I've attached the screenshot and the input spectrum. The formula for equivalent width is  at https://en.wikipedia.org/wiki/Equivalent_width. I think for this calculation one should do sums rather than a fancy numerical integral or interpolation.

<img width="1427" alt="screen shot 2016-10-28 at 11 38 37 am" src="https://cloud.githubusercontent.com/assets/2261329/19812438/7a083970-9d03-11e6-922d-c68bb9ff182d.png">

[VIPERS_110055275_corr.txt](https://github.com/spacetelescope/specviz/files/558790/VIPERS_110055275_corr.txt)

**Comment by nmearl** [comment]
What loader are you using for this data?

**Comment by pllim** [comment]
The EW algorithm (https://github.com/spacetelescope/specviz/blob/master/specviz/analysis/statistics.py#L83) was written by @stscieisenhamer, so maybe he can comment.

PySynphot also has a way to calculate EW (https://pysynphot.readthedocs.io/en/latest/properties.html#pysynphot-formula-equvw). You can probably hack PySynphot to calculate it by doing something like this:

```python
import pysynphot as S

wave, flux = extract_roi(your_spectrum)  # this is just pseudocode
df = 1 - flux / max(flux)
bp = S.ArrayBandpass(wave, df)
ew = bp.equivwidth()
```

**Comment by hcferguson** [comment]
The PySynphot formula is assuming the continuum is the maximum value in the ROI. I don't think we want to do that. First off, it will give the wrong answer for the EW of emission lines. Secondly, we are defining three ROIs for the EW measurement in specviz, and using the two on either side of the feature to estimate the continuum (as the mean flux in the two ROIs).

**Comment by pllim** [comment]
Well, you can adjust the second line to whatever formula you want. My point is once you normalize the flux properly, you can trick PySynphot to calculate EW for you. I am just suggesting it in case you cannot find a better solution.

**Comment by nmearl** [comment]
Is this more reasonable?

![screen shot 2016-10-28 at 1 24 45 pm](https://cloud.githubusercontent.com/assets/4141126/19815664/ecc94b08-9d11-11e6-8344-7088dc66f066.png)

**Comment by hcferguson** [comment]
Yes. Equivalent widths are usually reported in Angstroms, which is what I think you are doing here. We'll have to decide what to do when the X-axis is in different units. I tend to think we should stick to Angstroms. But that means we should probably report the units to avoid confusion.

**Comment by nmearl** [comment]
Currently, the EW is calculated and displayed in whatever units the plot is currently in. However, that's probably best for another issue. This issue is solved with afc971db04da24a94500a8e5100cb38e01d874e2.
